### DISCUSSION TITLE
Knife4j v5.0版本

### DISCUSSION THREAD

**Discussion by xiaoymin** [xiaoymin/knife4j#557] [Ideas]
未来 5.0的一些规划和畅想

**Comment by xiaoymin** [comment]
目前的Ui的一些功能整理，接下来的方向目标确定

<img width="1467" alt="Knife4jUi" src="https://user-images.githubusercontent.com/7894406/230042950-436c23d3-2653-49d1-88f4-76d005e6b9d2.png">

**Comment by leonlau81** [comment]
还会继续更新么，4.5.0版本发布后，已经快一年半没更新了
